### PR TITLE
External flow: standardize `empty.model.yml`

### PR DESCRIPTION
**Pull request by owen-mc**

### CODE REVIEW

**Review by michaelnebel — DISMISSED** [review]
LGTM

File: csharp/ql/lib/ext/empty.model.yml [hunk 1]
```diff
@@ -1,4 +1,6 @@
 extensions:
+  # Make sure that the extensible model predicates have at least one definition
+  # to avoid errors about an undefined extensionals.
```
Comment (hvitved) [inline csharp/ql/lib/ext/empty.model.yml:3]: Remove 'an'?

**Review by atorralba — COMMENTED** [review]
Java LGTM + what @hvitved said.

**Comment by asgerf** [comment]
LGTM assuming the fix from @hvitved is in

**Review by mbg — APPROVED** [review]

**Review by tausbn — APPROVED** [review]
Python 👍

**Review by hvitved — APPROVED** [review]

**Review by atorralba — APPROVED** [review]

**Pull request merged**